src/main/resources/META-INF/plugin-shell.xml

```diff
@@ -0,0 +1,6 @@
+<idea-plugin>
+    <extensions defaultExtensionNs="com.intellij">
+        <multiHostInjector order="first"
```

Comment:
`order=first` to prioritize JetBrains Shell if both plugins are enabled.
As the vendor of BashSupport Pro, I'd like to prefer BashSupport because it's unlikely that a user wants JetBrains Shell if there's a valid license of BashSupport Pro, but I wanted to keep the old behavior.